schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -623,6 +623,7 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.ServiceBus.json#/resourceDefinitions/namespaces_migrationConfigurations" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_AuthorizationRules" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_NetworkRuleSets" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_NetworkRuleSets" }, [](start = 18, length = 141)

we recently added a new root schema: 2019-04-01/deploymentTemplate.json. Can you please add the entry to that root schema too?

Reply:
Resolved. Added in  2019-04-01/deploymentTemplate.json